.github/workflows/devpackpublish.yml

```diff
@@ -23,7 +23,7 @@ jobs:
           token: ${{ secrets.REPO_ACCESS_TOKEN }}
       - uses: actions/setup-node@v3
         with:
-          node-version: 14
+          node-version: 16
```

Comment:
I enjoy [this feature of setup-node](https://github.com/BTE-Trusted-Entity/sporran-extension/blob/main/.github/workflows/pull_request.yaml#L11-L13) a lot

Reply:
this is not precise though, our version is set to >=16. What we could do is reference a specific lts in the .nvmrc and use that?

Reply:
I believe it would have worked correctly with package.json as well, but since you already have .nvmrc this would be the best.